### ISSUE TITLE
[Bug]: Invalid option(s) "queue_name" passed to the Redis Messenger transport.

### ISSUE THREAD

**Issue by matthiashamacher** [pimcore/pimcore#11355]
### Expected behavior

Switch the transports (Redis, In Memory, etc) for the messenger component

### Actual behavior

The problem is, that with 10.3 there was a messenger configuration "queue_name" introduced that is only available if you use Doctrine as transport. In case you use other transports, Redis in my example, everything fails because the option is not available in the Redis configuration. Therefore you need to switch to Doctrine or create your own transports without the option to fix the problem.

### Steps to reproduce

1. Install Redis
2. Configure Redis as transport
3. Try to save something in the backend or run the command `bin/console messenger:consume pimcore_core pimcore_maintenance` 

I don't know if it was intended that only doctrine is working out of the box from now on but at least a hint the Upgrade Notes would be helpful, that you need to configure completely new transports if you want to use other systems.

**Comment by fashxp** [comment]
responsible PR was https://github.com/pimcore/pimcore/pull/10928 ...
and no, it is not intentional, but the transports were configured for doctrine before, so some custom configuration is needed for sure to support different transports, isn't it?

**Comment by matthiashamacher** [comment]
Yes. But previously I only needed to change the DSN which was only this code:
```
framework:
    messenger:
        transports:
            pimcore_core: '%env(resolve:REDIS_MESSENGER_DSN)%/core'
            pimcore_maintenance: '%env(resolve:REDIS_MESSENGER_DSN)%/maintenance'
```

Now, as I cannot use these transports any more I need to configure my own transports an override the default routing config.

```
framework:
    messenger:
        transports:
            pimcore_core_redis:
                dsn: '%env(resolve:REDIS_MESSENGER_DSN)%/core'
            pimcore_maintenance_redis:
                dsn: '%env(resolve:REDIS_MESSENGER_DSN)%/maintenance'
        routing:
            'Pimcore\Messenger\SendNewsletterMessage': pimcore_core_redis
            'Pimcore\Messenger\VideoConvertMessage': pimcore_core_redis
            'Pimcore\Messenger\CleanupThumbnailsMessage': pimcore_core_redis
            'Pimcore\Messenger\SearchBackendMessage': pimcore_core_redis
            'Pimcore\Messenger\SanityCheckMessage': pimcore_core_redis
            'Pimcore\Messenger\AssetUpdateTasksMessage': pimcore_core_redis
            'Pimcore\Messenger\GeneratePagePreviewMessage': pimcore_core_redis
            'Pimcore\Messenger\GenerateWeb2PrintPdfMessage': pimcore_core_redis
            'Pimcore\Messenger\VersionDeleteMessage': pimcore_core_redis
            'Pimcore\Messenger\OptimizeImageMessage': pimcore_core_redis
            'Pimcore\Messenger\MaintenanceTaskMessage': pimcore_maintenance_redis
```

This works, but it is not clear, that it needs to be done. That's because I wrote that maybe just information in the Release Notes already helps.

**Comment by dvesh3** [comment]
maybe defining it like below in the core helps:
```yaml
messenger:
        transports:
            pimcore_core: "doctrine://default?queue_name=pimcore_core"
            pimcore_maintenance: "doctrine://default?queue_name=pimcore_maintenance"
```
@matthiashamacher could you please give it a try? and provide a PR, if it works. thanks!

**Comment by matthiashamacher** [comment]
@dvesh3 Thanks, that works.

**Comment by dvesh3** [comment]
Fixed by #11369
